frontend/src/App.tsx

```diff
@@ -160,7 +164,7 @@ const App: Component = () => {
         </Show>
 
         <Show when={lobby()}>
-          <Lobby wiki={wiki} id={id} lobby={lobby} search={search} />
+          <Lobby players={players} wiki={wiki} id={id} lobby={lobby} search={search} />
```

Comment:
it would be cool if we could just import the signals from one place and not propagating them all the way down but thats problem a task for another refactoring